tests/sealKey/sealKey.edl

```diff
@@ -5,5 +5,20 @@ enclave {
     trusted {
         public int test_seal_key (
             int in);
+
+        public int enc_get_public_key_by_policy(
+            int policy,
+            [string, in] const char* data,
+            [out] uint8_t** keybuf,
```

Comment:
Avoid oe_host_malloc calls within the enclave. Instead use the following pattern:

[out, count=max_keybuf_length] uint8_t* keybuf,
size_t max_keybuf_length,
[out] size_t* key_buf_length